app/code/Magento/Captcha/Controller/Refresh/Index.php

```diff
@@ -18,12 +18,24 @@ class Index extends \Magento\Framework\App\Action\Action
     protected $captchaHelper;
 
     /**
+     * @var \Magento\Framework\Serialize\SerializerInterface
+     */
+    protected $serializer;
+
+    /**
      * @param Context $context
      * @param \Magento\Captcha\Helper\Data $captchaHelper
+     * @param \Magento\Framework\Serialize\SerializerInterface|null $serializer
+     * @throws \RuntimeException
      */
-    public function __construct(Context $context, \Magento\Captcha\Helper\Data $captchaHelper)
-    {
+    public function __construct(
+        Context $context,
+        \Magento\Captcha\Helper\Data $captchaHelper,
+        \Magento\Framework\Serialize\SerializerInterface $serializer = null
+    ) {
         $this->captchaHelper = $captchaHelper;
+        $this->serializer = $serializer ?: \Magento\Framework\App\ObjectManager::getInstance()
```

Comment:
* Please use `Magento\Framework\Serialize\Serializer\Json` instead of `SerializerInterface`, since we can not guarantee that preference will remain Json in the future.
* Since you are not modifying the existing constructor but adding a new one – you can inject the dependency in a normal way.